low-level/files/etc/logrotate.d/iem_messages

```diff
@@ -1,14 +0,0 @@
-/var/log/cortx/iem/iem_messages
```

Comment:
as @sandeepsgt identified, we will need IEM logging (& logrotate) related config still, not related to SSPL logging

Reply:
Addressed, please check.